TriangulationForm.html

```diff
@@ -537,7 +538,17 @@ <h4 class="panel-title">
 		}else if(obj.cohortName.includes("_LTFU")){
 			$("#obs\\.type_of_visit").val("2160^VISITA DE BUSCA^99DCT");
 			$('#obs\\.type_of_visit').prop('disabled', 'disabled');
+		}else if(obj.cohortName.includes("_AMA_LTFU")){
```

Comment:
All _*_LTFU will not be reached as the _LTFU on line 538 will be used. Instead have the else if with _SV and _LTFU as the last conditions